### PR TITLE
Filling out constants and some calibrations

### PR DESCRIPTION
**Pull request by balghane**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -118,28 +118,32 @@ public static final class SwerveDrive {
...
+    public static final double ELEVATOR_MOTOR_GEAR_RATIO = 29.51;
 
+    public static final double ELEVATOR_RIGHT_ABSOLUTE_ENCODER_RATIO = 26/28;
+    
     /* Scalar for elevator motor encoder from rotations to real inches */
-    public static final double ELEVATOR_MOTOR_ENCODER_SCALAR = Constants.PLACEHOLDER_DOUBLE;
-
-    /* Scalar for arm motor encoder from RPM to real degrees per seconds */
-    public static final double ARM_MOTOR_ENCODER_VELOCITY_SCALAR = Constants.PLACEHOLDER_DOUBLE;
+    public static final double ELEVATOR_MOTOR_ENCODER_SCALAR = ELEVATOR_MOTOR_GEAR_RATIO;
```
Comment (tails618) [inline src/main/java/frc/robot/Constants.java]: Units in variable name

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -118,28 +118,32 @@ public static final class SwerveDrive {
...
+    
+    /* Scalar for elevator motor encoder from RPM to real inches per seconds */
+    public static final double ELEVATOR_MOTOR_ENCODER_VELOCITY_SCALAR =
```
Comment (tails618) [inline src/main/java/frc/robot/Constants.java]: Units in variable name

**Review by tails618 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (balghane) [reply src/main/java/frc/robot/Constants.java]: Fixed

File: src/main/java/frc/robot/Constants.java (hunk 2 above)
Comment (balghane) [reply src/main/java/frc/robot/Constants.java]: Fixed

**Review by tails618 — APPROVED** [review]

**Pull request merged**